### PR TITLE
Added new assets route with new schema addition

### PR DESCRIPTION
**Pull request by xunchuan-liu**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -400,6 +402,119 @@ mod tests {
         assert_eq!(another_value.serial_number, resp[2].serial_number);
     }
```
Comment (xunchuan-liu) [inline src/main.rs:404]: Added standard tests for this new route as well

File: migrations/2021-02-01-233741_create_assets/up.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-- Your SQL goes here
```
Comment (xunchuan-liu) [inline migrations/2021-02-01-233741_create_assets/up.sql:1]: Added a new migration to `diesel` for this new table

**Comment by xunchuan-liu** [comment]
I'm not sure what happened with the build failure from the log output. Any insights on fixing it would be appreciated.

File: src/main.rs [hunk 3]
```diff
@@ -400,6 +402,119 @@ mod tests {
...
+            .uri(format!("/assets/id/{}", id).as_str())
+            .header(
+                header::AUTHORIZATION,
+                format!("Bearer {}", ADMIN_USER.token),
+            )
+            .to_request();
+        let resp: assets::Asset = test::read_response_json(&mut app, req).await;
+        assert_eq!(id, resp.id);
+        assert_eq!(value_updated.asset_tag_id, resp.asset_tag_id);
+
```
Comment (trueb2) [inline src/main.rs:494]: Seems to be missing the matching request to `#[get("/assets/asset_tag/{id}")`

**Review by trueb2 — APPROVED** [review]
I am okay with this approach. The Asset is decoupled from the the tagging of the asset. The asset tag may be decoupled from the physical tagging based on the usage of the endpoint. Querying against metadata on AssetTags will provide our analysis.

**Pull request merged**